sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py

```diff
@@ -53,8 +54,8 @@ def export(self, spans: Sequence[Span]) -> SpanExportResult:
             logger.exception("Exception occurred while exporting the data.")
             return get_trace_export_result(ExportResult.FAILED_NOT_RETRYABLE)
 
-    def shutdown(self):
-        """Shuts down the exporter.
+    def close(self) -> None:
```

Comment:
`shutdown` is the method that is inherited from [SpanExporter](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py#L62) in Opentelemetry. We should stick to the same method name.